server/journeys/referral/details/view.njk

```diff
@@ -66,7 +77,7 @@
         }) }}
         {{ govukSelect({
             label: {
-                html: '<h2 class="govuk-heading-m">What was the incident type?</h2>'
+                html: incidentTypeLabel
```

Comment:
Should we be passing this in as text and letting the component do the formatting for us? With `classes: "govuk-label--m"`

Reply:
Good point, done and pushed.